README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.com/ljte/lab-project.svg?branch=master)](https://travis-ci.com/ljte/lab-project) [![Coverage Status](https://coveralls.io/repos/github/ljte/lab-project/badge.svg?branch=master)](https://coveralls.io/github/ljte/lab-project?branch=working_branch)
+
```

Comment:
(https://coveralls.io/github/ljte/lab-project?branch=working_branch) - should be master

Reply:
done